### ISSUE TITLE
Player was able to join password-protected server

### ISSUE THREAD

**Issue by ultradm** [anegostudios/VintageStory-Issues#2810]
### Game Version

v1.18.5

### Platform

Windows

### Modded

Modded

### SP/MP

Multiplayer

### Description

Someone was able to join my password-protected multiplayer server despite it having a strong password. No idea how it had happened, and I can't reproduce it. I just thought I'd report it in case other users have had the same problem.

**Comment by ultradm** [comment]
> Have you checked with other players to see if anyone shared the password?

Yup I did. It wasn't shared with anyone at all.

However, I have restarted the server and updated it multiple times over the last few months, so I'm not sure if, in the course of doing so, the server password was ever deactivated. And I foolishly went to set the password right after this happened, so I have no way of verifying if it indeed was deactivated. Sigh.

**Comment by ultradm** [comment]
> Have you checked with other players to see if anyone shared the password?

Yup I did. It wasn't shared with anyone at all.

However, I have restarted the server and updated it multiple times over the last few months, so I'm not sure if, in the course of doing so, the server password was ever deactivated. And I foolishly went to set the password right after this happened, so I have no way of verifying if it indeed was deactivated. Sigh.
